db/seed.ts

```diff
@@ -1,19 +1,26 @@
 import { exit } from 'process';
 import { db } from '@/database.providers';
 import { users } from '@/users/users.model';
+import { faker } from '@faker-js/faker';
 
-const data = {
-  id: 1, //do not use 'users.id.default', seed should be idempotent
-  email: 'test@email.com',
-  username: 'test',
-  password: 'test',
-  bio: 'test',
-  image: 'test',
-};
-console.log('Upserting user: ', data);
-await db.insert(users).values(data).onConflictDoNothing();
-console.log('User upserted');
+console.log('Truncating the user database');
+await db.delete(users);
+console.log('The database is empty: ', await db.select().from(users));
 
+for (let i = 0; i < 10; i++) {
+  const data = {
+    id: faker.datatype.number(),
+    email: faker.internet.email(),
+    username: faker.internet.userName(),
+    password: faker.internet.password(),
```

Comment:
The password must be hashed. See:
https://github.com/agnyz/elysia-realworld-example-app/blob/6753ffd788e5013f76f514c683acf12822e7d4bb/src/users/users.service.ts#L46

Reply:
Thanks! I added hashing

Reply:
Well done!